apps/admin-gui/src/app/admin/pages/admin-page/admin-consent-hubs/admin-consent-hubs.component.ts

```diff
@@ -31,4 +44,26 @@ export class AdminConsentHubsComponent implements OnInit {
       this.loading = false;
     });
   }
+
+  evaluateConsents(): void {
+    const config = getDefaultDialogConfig();
+    config.width = '500px';
+    config.data = {
+      theme: 'user-theme',
+      message: this.translate.instant('ADMIN.CONSENT_HUBS.CONFIRM_EVALUATION'),
+    };
+
+    const dialogRef = this.dialog.open(UniversalConfirmationDialogComponent, config);
```

Comment:
Also I messed up with my advice, the universalConfirmation dialog is ugly and non optimal for this case
You need to adjust UniversalRemoveItemsDialog - Rename it to universalConfirmationDialog and then do two more 
Input() option to that dialog - first make an option to show the bold question and second for if the submit button will have the green on red color

![image](https://user-images.githubusercontent.com/32871167/166456689-6a0ad00d-7302-4285-9a69-2bdd743931bf.png)

The title of this specific dialog could be something like - "Evaluate consents confirmation" the subtitle can be something you already wrote down and then you will show in list the consent hubs